### PR TITLE
Add marker IS_WORKER to global settings

### PR DESCRIPTION
**Pull request by critias**

### CODE REVIEW

**Comment by JackTemaki** [comment]
I think this will work, but I am not sure if putting it into `global_settings.py` without a comment is a good idea, because people might think that this is a normal settings parameter....

**Comment by curufinwe** [comment]
I agree with @JackTemaki . Maybe this can be a property of the job, i.e. `sis_in_worker`.

**Comment by JackTemaki** [comment]
In the Job itself is maybe not a good idea, because I want to use this also outside of jobs... But I guess if this is a static variable in the Job class there can be some "helper" function to call it...

**Comment by critias** [comment]
You are right global_settings.py was a bad place, but I don't like to attach it to the job class. The toolkit looks like a good place to me.

**Review by JackTemaki — APPROVED** [review]

**Comment by JackTemaki** [comment]
Ah I was too fast, the `global_settings.py` edit is still in there, which needs to be removed...

File: sisyphus/global_settings.py [hunk 1]
```diff
@@ -354,6 +354,8 @@ def update_global_settings_from_file(filename):
     update_global_settings_from_text(content, filename)
 
 
+IS_WORKER = False
```
Comment (JackTemaki) [inline sisyphus/global_settings.py:357]: This needs to be removed

**Review by JackTemaki — CHANGES_REQUESTED** [review]

File: sisyphus/worker.py [hunk 2]
```diff
@@ -153,6 +154,8 @@ def stop(self):
 def worker(args):
     # Change job into error state in case of any exception
     gs.active_engine = gs.engine().get_used_engine(args.engine)
+    sisyphus.toolkit._sis_running_in_worker = True
```
Comment (curufinwe) [inline sisyphus/worker.py:157]: Technically I guess we should also set this back to false in a finally block afterwards.

Comment (critias) [reply sisyphus/worker.py:157]: I don't think it's relevant at the moment, but you are right. I added it.

**Review by JackTemaki — APPROVED** [review]

**Pull request merged**